src/qp_klp/Assays.py

```diff
@@ -166,6 +167,7 @@ def execute_pipeline(self):
             self.convert_raw_to_fastq()
             self.integrate_results()
             self.generate_sequence_counts()
+            self.subsample_reads()
```

Comment:
Does this mean that we will now ALWAYS subsample?

Reply:
Yes, but really we will always check if subsample is needed and only run it when necessary.

Reply:
Well, we will always subsample every fastq that has more than the max number of reads, right?